src/rust/plugin-work-queue/tests/psql_queue_tests.rs

```diff
@@ -0,0 +1,102 @@
+#![cfg(feature = "integration")]
+
+#[cfg(test)]
+mod tests {
+    use chrono::prelude::*;
+    use plugin_work_queue::psql_queue::{
+        get_status,
+        ExecutionId,
+        PsqlQueue,
+        Status,
+    };
+    use sqlx::postgres::PgPoolOptions;
+
+    #[derive(Debug)]
+    pub struct ExecutionRequest {
+        execution_key: ExecutionId,
+        plugin_id: uuid::Uuid,
+        pipeline_message: Vec<u8>,
+        execution_result: Option<Vec<u8>>,
+        status: Status,
+        creation_time: DateTime<Utc>,
+        last_updated: DateTime<Utc>,
+        visible_after: Option<DateTime<Utc>>,
+        try_count: i32,
+    }
+
+    #[tokio::test]
+    async fn test_migrate() -> Result<(), Box<dyn std::error::Error>> {
+        let pool = PgPoolOptions::new()
+            .max_connections(1)
+            .connect("postgresql://postgres:password@localhost:5432/postgres")
```

Comment:
The fact this communicates to localhost HEAVILY implies to me that this test is not hooked up correctly.

Sure enough, it looks like `RUST_INTEGRATION_TEST_FEATURES=` was never updated to include these tests.

Reply:
I suspect you're running this locally on your box with some form of `cargo test`. That style of development is convenient but should be considered deprecated or, worse, misleading.

Reply:
Yeah, the test_migrate needs to die entirely actually. As for adding the tests, will do. And I'll have this use a from_env kinda thing.